test/python/transpiler/test_remove_diagonal_gates_before_measure.py

```diff
@@ -0,0 +1,369 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2019, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""Test RemoveDiagonalGatesBeforeMeasure pass"""
+
+import unittest
+
+from qiskit import QuantumRegister, QuantumCircuit, ClassicalRegister
+from qiskit.transpiler import PassManager, transpile
```

Comment:
can you use `transpile` from `qiskit.compiler`, as the one from `qiskit.transpiler` is deprecated.

Reply:
Done in f1659fe6